### PR TITLE
Bugfix: Determination creation UI does not use default messages

### LINKED ISSUE #2455: Determination Templates Not Implemented
**Describe the bug**

There is functionality for the user to add determination message templates but those templates are not imported when the user creates a determination for an application. 

**To Reproduce**
1. Create determination template messages.
2. Add a determination to an application.

**Expected behavior**

I expect that those determination template messages would populate the text editor. 

**Priority**

- Low priority (annoying, would be nice to not see)

**Affected roles**

- Staff
- Reviewers

**Additional context**
Patch inbound.

### PR DESCRIPTION
**Pull request by hawkinsw**
In Hypha, the user has the ability to create default determination
messages. These messages were not being properly populated in the UI
shown when creating a determination for an application. This change
fixes that bug.

Fixes #2455

### CODE REVIEW

**Review by kfogel — CHANGES_REQUESTED** [review]
In the PR, you mention that this fixes bug #2455.  The commit message needs to say that too, though!  :-)

File: hypha/apply/determinations/templates/determinations/base_determination_form.html [hunk 1]
```diff
@@ -61,5 +69,5 @@ <h1>message</h1>
 {% endblock %}
 
 {% block extra_js %}
-    {# Skip this until the script is improved. <script src="{% static 'js/apply/determination-template.js' %}"></script> #}
+    <script src="{% static 'js/apply/determination-template.js' %}"></script>
```
Comment (kfogel) [inline hypha/apply/determinations/templates/determinations/base_determination_form.html:72]: What a satisfying part of the diff to see :-).

File: hypha/apply/determinations/views.py [hunk 2]
```diff
@@ -279,6 +279,7 @@ def get_object(self, queryset=None):
         return self.model.objects.get(submission=self.submission, is_draft=True)
 
     def dispatch(self, request, *args, **kwargs):
+
```
Comment (kfogel) [inline hypha/apply/determinations/views.py:282]: Unrelated whitespace change.  No big deal, and certainly not worth holding anything up about, but in general those are good to avoid in a code diff (at least IMHO).

**Comment by frjo** [comment]
@hawkinsw I have added you to the Hypha repo (you as well @kfogel). Can you move this PR to the Hypha repo so we can run all the tests etc.?

And thanks for fixing this feature!

**Comment by hawkinsw** [comment]
Closing this and reopening one from a branch within the hypha repo!
